src/components/Dashboard/Alerts/NotInCompliance.vue

```diff
@@ -0,0 +1,97 @@
+<template>
+  <v-expansion-panels v-model="panel" id="not-in-good-standing-container">
+    <v-expansion-panel class="mb-6">
+      <v-expansion-panel-header hide-actions class="d-flex justify-space-between px-6 py-5">
+        <h3>
+          <v-icon left color="orange darken-2">mdi-alert</v-icon>
+          <span>This business is not in compliance</span>
+        </h3>
+        <v-btn text color="primary" class="details-btn my-n1" @click.stop="togglePanel()">
+          <span color="primary">{{ panel === 0 ? "Hide Details" : "View Details" }}</span>
+          <v-icon right color="primary">
+            {{ panel === 0 ? "mdi-chevron-up" : "mdi-chevron-down" }}
+          </v-icon>
+        </v-btn>
+      </v-expansion-panel-header>
+
+      <v-expansion-panel-content>
+        <!--
+          FUTURE: Add helpful text here. See NotInGoodStanding.vue for sample layout.
+        -->
```

Comment:
Waiting for feedback from Yui.

Reply:
The BE doesn't provide compliance warnings yet, so this code is not used at the moment. Hence, it's for future use. I recommend you just review that it looks OK and don't worry about the missing content.